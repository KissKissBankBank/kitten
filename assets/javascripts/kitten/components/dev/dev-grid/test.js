import React from 'react'
import renderer from 'react-test-renderer'
import { DevGrid } from './index'

describe('<DevGrid />', () => {
  let component

  describe('with default props', () => {
    beforeEach(() => {
      component = renderer.create(<DevGrid visible />).toJSON()
    })

    it('matches with snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
