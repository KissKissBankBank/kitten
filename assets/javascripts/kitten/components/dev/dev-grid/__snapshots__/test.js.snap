// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DevGrid /> with default props matches with snapshot 1`] = `
<div
  className="sc-bdVaJa fBOtcW k-Dev"
>
  <div
    className="k-DevBreakpoint"
  />
  <div
    className="k-DevGrid"
  >
    <div
      className="k-DevGrid__container"
    >
      <div
        className="k-DevGrid__row"
      >
        <div
          className="k-DevGrid__col"
        >
          <div
            className="k-DevGrid__inner"
          />
        </div>
        <div
          className="k-DevGrid__col"
        >
          <div
            className="k-DevGrid__inner"
          />
        </div>
        <div
          className="k-DevGrid__col"
        >
          <div
            className="k-DevGrid__inner"
          />
        </div>
        <div
          className="k-DevGrid__col"
        >
          <div
            className="k-DevGrid__inner"
          />
        </div>
        <div
          className="k-DevGrid__col"
        >
          <div
            className="k-DevGrid__inner"
          />
        </div>
        <div
          className="k-DevGrid__col"
        >
          <div
            className="k-DevGrid__inner"
          />
        </div>
        <div
          className="k-DevGrid__col"
        >
          <div
            className="k-DevGrid__inner"
          />
        </div>
        <div
          className="k-DevGrid__col"
        >
          <div
            className="k-DevGrid__inner"
          />
        </div>
        <div
          className="k-DevGrid__col"
        >
          <div
            className="k-DevGrid__inner"
          />
        </div>
        <div
          className="k-DevGrid__col"
        >
          <div
            className="k-DevGrid__inner"
          />
        </div>
        <div
          className="k-DevGrid__col"
        >
          <div
            className="k-DevGrid__inner"
          />
        </div>
        <div
          className="k-DevGrid__col"
        >
          <div
            className="k-DevGrid__inner"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
